Create WEP0002-slack-community-moderation-procedures.rst (#4)
writethedocs/weps: drafts/WEP0002-slack-community-moderation-procedures.rst

* **Take care of yourself and your mental health** - this community contains a lot of great people who are great at self-moderating, and while you are partly responsible for keeping things running smoothly, your wellbeing should always take priority. If you need to take a break, please do. You're human, you have work and other life responsibilities to worry about, and you can't take on everything yourself. You can always ask in the #slack-mods channel for someone else to take over a particular situation.

At the end of the day, we're all human and we all make mistakes that we then learn from. The goal here isn't perfect behaviour, but to ensure that we set an example for the rest of the community in terms of treating each other well, and being able to disagree with each other respectfully.

Post removal
------------
When a user posts obvious spam or posts with objectionable content, any one moderator is free to remove the material upon seeing it. Before removing the message, they should:

1. Share the message with the #slack-mods channel to notify the rest of the moderators of what happened, and to keep a record of the deleted message.

> 2. Share the message with the poster to reference why the material was removed, and offer more information (including a link to the Code of Conduct) if necessary.

Reviewer: ```suggestion
2. Send a note to the poster referencing why the material was removed, and offer more information (including a link to the Code of Conduct) if necessary.
```

When a moderator removes a post, they may also decide to remove other users' responses. Sometimes this decision is appropriate given the context, and sometimes the discussion that follows (often involving community self-moderation) is worth keeping so members' responses can be visible.

> There's a certain amount of personal discretion that goes into removing posts, but when in doubt, the moderation team should generally refrain from removing posts. A lack of discretion in removing posts can negatively impact the community's trust in the moderation team.

Reviewer: ```suggestion
There's a certain amount of personal discretion that goes into removing posts. **When in doubt, the moderation team should generally refrain from removing posts.** A lack of discretion in removing posts can negatively impact the community's trust in the moderation team.
```

* **Recruiter spam** - while it's not inherently bad for recruiters to find candidates within the Slack community, we want to make sure they don't detract from it in doing so. Spamming the community with job postings (and thereby distracting from posts from users who are affiliated with the postings) and pushy DMs give the impression that they're trying to mine the community for financial reasons, and that's not what we want. Because moderators can't monitor DMs, we depend on users to speak up when they happen in order to take action, and should communicate clearly that they can and should.

> * **Marketing posts in #intros** - the purpose of the #intros channel is to get to know new members of the community. It's often a difficult line to draw between a person and their work, but a general rule of thumb is that users shouldn't be looking for sales prospects in that channel. Posts including things like personal websites and social media links with invitations to connect are okay; however, writing on behalf of a business looking for potential clients would fall under sales-y content and thus would be subject to removal. At that point, a moderator can send the user a message explaining the removal, along with a warm invitation to post again without the sales angle.

Reviewer: > a moderator can send the user a message explaining the removal, along with a warm invitation to post again without the sales angle.

I'd love to see some templated responses for common situation like this, along with the 3 strikes rule. It doesn't need to live in this document, but having a way to accumulate these templates over time will be useful.

Author: Just added a sample script! I think having everything in one place makes sense.

The only thing I'm still unsure of is whether I can format the block quote to look like ones between paragraphs rather than between bullet points, but either way, the content is there.

With some of these rules, we've been more lenient towards long-time contributors to the community with whom we've built trust over time. In those cases, we can trust that the community will recognize them and know they're not looking for business prospects. If you're unsure, searching through the user's past posts or checking out their profile can offer clues about other ways in which they've engaged with the community and whether they're affiliated with a product. Some detective work can be useful in identifying whether a complimentary post is sales-y or an honest endorsement.

With these better defined post guidelines in place, members of the Slack community can feel confident that they can interact with each other as individuals and as professionals, rather than as business prospects, and moderators can more easily remove roadblocks to that end.

> Code of conduct violations

Reviewer: I feel like this might be better under a heading of "Three strikes rule" or similar, as this is a bit broader than explicit COC violations.

Author: It doesn't add much, so I opted to just take it out instead of letting it get in the way. :)

--------------------------

> Historically, the moderation team has struggled to consistently handle code of conduct violations. Some of the problems we faced included:

Reviewer: ```suggestion
Historically, the moderation team has struggled to consistently handle Code of Conduct violations. Some of the problems we faced included:
```

Regardless of which action is taken, every time a moderator needs to take action, they should let the other moderators know and track it in the tracking spreadsheet. For privacy reasons, this spreadsheet should only be accessible to the moderation group (currently, the link to the spreadsheet is pinned in the #slack-mods channel, but we may want to change viewing/editing permissions to only people who are invited, rather than anyone who has the link). The spreadsheet can help the moderation team ensure that we're consistent in the types of conduct we respond to and our responses to it. We could consider an expiration date of, say, a year, at which point actions "expire" and won't be taken into account anymore - making an assumption that people learn over time, and won't have mistakes count against them forever.

> Depending on how visible the user's conduct was to the greater community, it may make sense to communicate moderation actions to the greater community (in the #announcements channel, or just in the channel the conversation(s) occurred in), both for transparency and to reassure community members that the moderation team is willing to act if they need us to.

Reviewer: I think we should probably not mention posting something in `#announcements`. I feel like that's a pretty passive aggressive way of handling these things, and I don't know where it would be appropriate instead of following up in the channel where things happened. 

Perhaps posting in `#meta`, or some kind of public moderation channel would be a good middle ground here.

Author: Sure, that's fine - I only brought up the possibility because we'd discussed it in the slack mod channel before. The reasoning there was that there was no way for people to opt out of that channel, so we'd make sure that everyone would see a message like "racism is bad, don't do it."

1. Verbal reminder
__________________

> The first time a member of the moderation team contacts a user, or for constact over minor actions, it can be an in-channel reply or DM; at this point, we can assume that the user may not be cognisant of the expectations in the community.

Reviewer: Not 100% sure this is the right edit. 

```suggestion
The first time a member of the moderation team contacts a user, or for multiple minor actions, it can be an in-channel reply or DM; at this point, we can assume that the user may not be cognisant of the expectations in the community.
```

Author: Whoops, thanks for catching that typo! I meant "contact," which could just happen once.

An example of a more formal warning in DM format:

  Hi Mildred,

  I'm Janine, one of the moderators here in the Write the Docs Slack community. I wanted to let you know that your conversation in #general earlier today violated our Code of Conduct rules, specifically the one that asks all users to remain respectful of each other at all times.

  While I'm sure you were operating in good faith, we want to ensure that all users in this Slack community feel comfortable engaging with each other, and this means making sure that we all act civilly. To this end, I wanted to give you a reminder that we need you to follow those rules at all times.

  If you have any questions, please don't hesitate to reach out and ask - the moderators are here to help. Thanks!

2. Formal warning
________________
If a user continues to violate the Code of Conduct, a moderator can give them a more stern warning about their conduct. Example wording:

  Hi Mildred,

  I wanted to check in with you again regarding your posts in #general. As discussed before, I want to make it clear that all members of this community are required to follow the Code of Conduct rules at all times, with no exceptions.

>   If this continues to be a problem, the moderation team will have no choice but to consider further action up to and including removing you from the Write the Docs community the next time you violate the rules of this community. Because this is always a last resort that the moderation team wants to prevent wherever possible, I wanted to give you a fair warning first and the opportunity for you to ask me questions if you need to. Thanks!

Reviewer: ```suggestion
  This is a formal warning from the moderation team. If this continues to be a problem, the moderation team will have no choice but to consider further action up to and including removing you from the Write the Docs community the next time you violate the rules of this community. Because this is always a last resort that the moderation team wants to prevent wherever possible, I wanted to give you a fair warning first and the opportunity for you to ask me questions if you need to. Thanks!
```

3. Removal from the community
_____________________________
As a last resort, if a user has proven to be unable or unwilling to follow the Code of Conduct, it's in the best interest of the community for moderators to remove the user. This decision should never be taken lightly, and whenever possible, it should always be made in collaboration with other moderators. (If a user is being particularly aggressive, it might be in the community's best interest for a moderator to take action more immediately, but this should always be communicated to the rest of the moderators as soon as possible.)

> Depending on the timing/circumstances (e.g., moderators may be concerned about the user lashing out), this message may be delivered via DM or to the email address attached to the user's account. Example wording:

Reviewer: I think we should specify that this is always sent via email, given the fact we're removing them from Slack :)

* **Feel free to express personal opinions civilly** - being a moderator doesn't mean you aren't a valued member of the community with valuable experience to share. That said, you are always expected to set a good example. Disagreements are fine, but express them respectfully. If you feel that can't do that, then remove yourself from the conversation. If needed, you can tag in another moderator to take over for you if one is available.
* **Identify yourself in your profile with a moderator symbol (the shield emoji has been suggested before)** - with the caveat that non-moderators can easily copy this, but trusting that they probably won't.
* **Wade into contentious conversations carefully, and prioritize de-escalation over expressing personal opinions** - by the time a conversation becomes heated, if a moderator is available, they should try to de-escalate the situation, including asking users to drop the conversation. This can be really hard, especially if the conversations revolve around sensitive topics. Take a deep breath, do what you can, and don't be afraid to disengage.

> * **Take care of yourself and your mental health** - this community contains a lot of great people who are great at self-moderating, and while you are partly responsible for keeping things running smoothly, your wellbeing should always take priority. If you need to take a break, please do. You're human, you have work and other life responsibilities to worry about, and you can't take on everything yourself. You can always ask in the #slack-mods channel for someone else to take over a particular situation.

Reviewer 2: This is the first of several mentions of #slack-mods. I'd like to see this change throughout to something like "the private moderation channel". I think referring to #slack-mods might give the (false) impression that there will be a public Slack moderation channel (apart from #meta), where the moderation deliberations will be publicly discussed.

Author: Done!

1. Share the message with the #slack-mods channel to notify the rest of the moderators of what happened, and to keep a record of the deleted message.
2. Send a note to the poster referencing why the material was removed, and offer more information (including a link to the Code of Conduct) if necessary.

When a moderator removes a post, they may also decide to remove other users' responses. Sometimes this decision is appropriate given the context, and sometimes the discussion that follows (often involving community self-moderation) is worth keeping so members' responses can be visible.

There's a certain amount of personal discretion that goes into removing posts. **When in doubt, the moderation team should generally refrain from removing posts.** A lack of discretion in removing posts can negatively impact the community's trust in the moderation team.

Sales-y content
---------------
Historically, the moderation team has struggled with the definition of "sales-y content" listed on the Slack guidelines. We currently define sales-y content as follows:

* Direct pitches
* Links without context
* Brands

Some further proposed definitions for the purpose of moderating are as follows:

> * **Unpublished market research** - being a helpful bunch, the Write the Docs community is never hesitant to weigh in on their experiences. However, market research that the community contributes to should also benefit the community. Therefore, we ask that by posting a survey and asking for responses, a user commits to allowing the community to access the results for free, and we reserve the right to decline or remove a survey if we don't think that it'll benefit the community sufficiently. We could also create a #surveys channel so users who aren't interested in surveys don't see them. (Academic surveys for a user's studies don't have to be held to the same standards, but a channel for surveys could also theoretically benefit users who are in school and are looking for information about the field.)

Reviewer 2: > We could also create a #surveys channel so users who aren't interested in surveys don't see them.

This seems like an oddly specific proposal mixed in with general guidelines. I think we ought to either have this channel and start using that practice, or don't; either way it's something we could drop from this document.

Author: Sure! This was basically just a brain dump of things I thought might be good ideas, but happy to remove it and pick up the discussion elsewhere if we need to.

> * **Unpublished market research** - being a helpful bunch, the Write the Docs community is never hesitant to weigh in on their experiences. However, market research that the community contributes to should also benefit the community. Therefore, we ask that by posting a survey and asking for responses, a user commits to allowing the community to access the results for free, and we reserve the right to decline or remove a survey if we don't think that it'll benefit the community sufficiently. We could also create a #surveys channel so users who aren't interested in surveys don't see them. (Academic surveys for a user's studies don't have to be held to the same standards, but a channel for surveys could also theoretically benefit users who are in school and are looking for information about the field.)

Reviewer 2: In general, I'm not quite sure what's going on with this point. The heading is about market research, but the text is all about surveys. As written, I worry that a) there's a non-survey research loophole, b) it's unclear about unpublished research or research published in subscription journals, c) and it leaves out unaffiliated researchers.

Perhaps we could reframe it to be less narrowly focused on surveys and to avoid the two-pronged academic/non-academic distinction?

Something like:

> Surveys, solicitations for interview subjects, and other research may be conducted via the Write the Docs Slack if it is accompanied by a commitment that the results of that research will contribute to the broader documentarian community. For example, a survey conducted with a pledge to make the results freely available or to use the data in a scholarly paper would satisfy this requirement.

That's not a complete rewrite—I really like and think we should retain the "we reserve the right to decline or remove [research efforts] if we don't think that it'll benefit the community sufficiently" text—but that's what I'm getting at here: we want to be open to stuff that advances the field as a whole (even though it might not directly benefit WTD specifically).

Author: Haha, yeah, this was in response to what I mostly remembered as a bunch of surveys that people were using for market research. I really like the idea of broadening the rule beyond surveys, and stole your wording :)
